<ui version="4.0" >
 <class>SlicerAngleForm</class>
 <widget class="QWidget" name="SlicerAngleForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>3dmod Slicer Angles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="setButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Set top slicer to angles and position into current row</string>
       </property>
       <property name="text" >
        <string>Set Angles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Delete the current row from the table</string>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel" >
       <property name="text" >
        <string>Time:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table" >
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount" >
      <number>7</number>
     </property>
     <column>
      <property name="text" >
       <string>X rot</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Y rot</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Z rot</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>X cen</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Y cen</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Z cen</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Label</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="volumeGroup" >
     <property name="title" >
      <string>Operations on volume angle sets</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="margin" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="removeButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Delete all rows for this time and shift times down for sets at higher time</string>
        </property>
        <property name="text" >
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insertButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Insert empty set of angles, shifting sets at this and higher times up one in time</string>
        </property>
        <property name="text" >
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Copy the set of angles from another time to this time</string>
        </property>
        <property name="text" >
         <string>Copy From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="copySpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Set the time to copy sets from</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="renumberButton" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Assign this angle set to a different time and shift other sets to fill gap</string>
        </property>
        <property name="text" >
         <string>Renumber To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="renumberSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>32767</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Set the time to which to assign sets</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="doneButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Close window</string>
       </property>
       <property name="text" >
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Open help window</string>
       </property>
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>helpButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>helpClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>setAngClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>copyClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renumberButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>renumberClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renumberSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>renumberChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>copyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insertButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>insertClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>SlicerAngleForm</receiver>
   <slot>removeClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
